<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>imageprocessing</class>
 <widget class="QWidget" name="imageprocessing">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1506</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Processing</string>
  </property>
  <widget class="QLabel" name="ImageDisplay">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>60</y>
     <width>853</width>
     <height>533</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="SelectImageFiles">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>201</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Image Files</string>
   </property>
  </widget>
  <widget class="QListWidget" name="InputList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>65</y>
     <width>301</width>
     <height>192</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="TerminalOut">
   <property name="geometry">
    <rect>
     <x>1210</x>
     <y>50</y>
     <width>281</width>
     <height>541</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ClearImageFiles">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>40</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="EncodeTest">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>10</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>1-bit -&gt; 24-bit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Input Images&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>1230</x>
     <y>10</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Terminal Window&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BrowseTargetDir">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>270</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Select Target Directory&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="TargetDirLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white; border: 1px solid black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Output Images&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QListWidget" name="OutputImageList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>301</width>
     <height>211</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="encodeProgress">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>20</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QPushButton" name="GreyscaleEncode">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>10</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>8-bit -&gt; 24-bit</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
